risteys_elixir/assets/css/tailwind.css

```diff
@@ -333,34 +334,29 @@ header .actions a {
  * PHENOCODE PAGE
  */
 
-/* Risteys-link */
-#Risteys-link {
```

Comment:
Here I preferred to add `ml-2` to an element that already had styling, giving the same result.